services/storage/src/simcore_service_storage/datcore_adapter/datcore_adapter.py

```diff
@@ -224,7 +233,7 @@ async def get_file_download_presigned_link(
         dict[str, Any],
         await _request(app, api_key, api_secret, "GET", f"/files/{file_id}"),
     )
-    return parse_obj_as(AnyUrl, file_download_data["link"])
+    return cast(AnyUrl, parse_obj_as(AnyUrl, file_download_data["link"]))  # mypy
```

Comment:
what is the `#mypy` comment for? Does mypy interpret it somehow?

Reply:
If it does not, I don't think it's that useful. I would already assume the casting is due to mypy.

I would just remove them all over the place where you added them.

Reply:
well to my understanding these casting are **only** due to mypy. they are semantically correct.
therefore the comment to explain why it is done this way.